scripts/bump-bb.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+BB_VERSION=$1
+
+sed -i "s/^VERSION=.*/VERSION=\"$BB_VERSION\"/" ./scripts/install_bb.sh
```

Comment:
This fails on Mac because `sed -i` works differently for GNU and Mac `sed`: https://stackoverflow.com/questions/7573368/in-place-edits-with-sed-on-os-x

```bash
❯ echo 'VERSION=9' > example_file
❯ BB_VERSION="10" sed -i "s/^VERSION=.*/VERSION=\"$BB_VERSION\"/" example_file
sed: 1: "example_file": invalid command code e
```

Reply:
```suggestion
sed -i.bak "s/^VERSION=.*/VERSION=\"$BB_VERSION\"/" ./scripts/install_bb.sh  && rm ./scripts/install_bb.sh.bak
```